### PR TITLE
✨ aws elb targetgroup

### PR DESCRIPTION
**Pull request by vjeffrey**
Screenshot 2024-05-27 at 21 08 00

### CODE REVIEW

File: providers/aws/resources/aws.lr [hunk 1]
```diff
@@ -1088,6 +1088,44 @@ aws.elb {
...
+  healthCheckPath string
+  // Health check port for the load balancer target group
+  healthCheckPort string
+  // Health check protocol for the load balancer target group
+  healthCheckProtocol string
+  // Health check timeout seconds for the load balancer target group
+  healthCheckTimeoutSeconds int
+  // Target type for the for the load balancer target group (instance, ip, lambda, alb)
+  targetType string
+  // Unhealthy treshold count for the load balancer target group
```
Comment (tas50) [inline providers/aws/resources/aws.lr]:
```suggestion
  // Unhealthy threshold count for the load balancer target group
```

File: providers/aws/resources/aws.lr.manifest.yaml [hunk 2]
```diff
@@ -1626,6 +1626,8 @@ resources:
       scheme: {}
       securityGroups:
         min_mondoo_version: 9.0.0
+      targetGroups:
+        min_mondoo_version: latest
```
Comment (tas50) [inline providers/aws/resources/aws.lr.manifest.yaml]:
```suggestion
        min_mondoo_version: 9.0.0
```

File: providers/aws/resources/aws.lr.manifest.yaml [hunk 3]
```diff
@@ -1635,6 +1637,32 @@ resources:
...
+      lambdaTargets: {}
+      name: {}
+      port: {}
+      protocol: {}
+      protocolVersion: {}
+      targetType: {}
+      unhealthyThresholdCount: {}
+      vpc: {}
+    is_private: true
+    min_mondoo_version: latest
```
Comment (tas50) [inline providers/aws/resources/aws.lr.manifest.yaml]:
```suggestion
    min_mondoo_version: 9.0.0
```

File: providers/aws/resources/aws.lr [hunk 4]
```diff
@@ -1088,6 +1088,44 @@ aws.elb {
...
+  // Target type for the for the load balancer target group (instance, ip, lambda, alb)
+  targetType string
+  // Unhealthy threshold count for the load balancer target group
+  unhealthyThresholdCount int
+  // VPC for the load balancer target group
+  vpc() aws.vpc
+  // EC2 targets for the load balancer target group
+  ec2Targets() []aws.ec2.instance
+  // Lambda targets for the load balancer target group
+  lambdaTargets() []aws.lambda.functio
```
Comment (tas50) [inline providers/aws/resources/aws.lr]:
```suggestion
  lambdaTargets() []aws.lambda.function
```

File: providers/aws/resources/aws.lr [hunk 5]
```diff
@@ -1088,6 +1088,44 @@ aws.elb {
...
+  healthCheckIntervalSeconds int
+  // Health check path for the load balancer target group
+  healthCheckPath string
+  // Health check port for the load balancer target group
+  healthCheckPort string
+  // Health check protocol for the load balancer target group
+  healthCheckProtocol string
+  // Health check timeout seconds for the load balancer target group
+  healthCheckTimeoutSeconds int
+  // Target type for the for the load balancer target group (instance, ip, lambda, alb)
```
Comment (tas50) [inline providers/aws/resources/aws.lr]:
```suggestion
  // Target type for the for the load balancer target group (instance, IP, Lambda, ALB)
```

File: providers/aws/resources/aws.lr [hunk 6]
```diff
@@ -1088,6 +1088,44 @@ aws.elb {
...
+  name string 
+  // ARN for the load balancer target group
+  arn string 
+  // Port for the load balancer target group
+  port int
+  // Protocol for the load balancer target group
+  protocol string
+  // Protocol version for the load balancer target group
+  protocolVersion string
+  // IP address type for the load balancer target group (ipv4, ipv6)
```
Comment (tas50) [inline providers/aws/resources/aws.lr]:
```suggestion
  // IP address type for the load balancer target group (IPv4, IPv6)
```

File: providers/aws/resources/aws.lr [hunk 7]
```diff
@@ -1088,6 +1088,44 @@ aws.elb {
   loadBalancers() []aws.elb.loadbalancer
 }
 
+// AWS ELB Target Group
+private aws.elb.targetgroup {
```
Comment (tas50) [inline providers/aws/resources/aws.lr]: targetgroup should go in the expect.txt file

File: providers/aws/resources/aws.lr [hunk 8]
```diff
@@ -1088,6 +1088,44 @@ aws.elb {
   loadBalancers() []aws.elb.loadbalancer
 }
 
+// AWS ELB Target Group
```
Comment (tas50) [inline providers/aws/resources/aws.lr]:
```suggestion
// AWS Elastic Load Balancer (ELB) Target Group
```

**Comment by tas50** [comment]
LGTM from the product side

**Review by imilchev — APPROVED** [review]

**Pull request merged**